test-e2e/requirements.txt

```diff
@@ -1,9 +1,9 @@
-git+https://github.com/dcos/dcos-e2e.git@2020.05.26.0
+git+https://github.com/dcos/dcos-e2e.git@2020.12.18.0
```

Comment:
With the new pip resolver, we needed a newer version of dcos-e2e which also supported `cryptography==3.2.1`

Reply:
Thanks for this update.